rootfs/bin/get_object

```diff
@@ -2,12 +2,10 @@
 
 GET_PATH=/tmp/slug.tgz
 
-export BUCKET_FILE=/var/run/secrets/deis/objectstore/creds/builder-bucket
 if [ "$BUILDER_STORAGE" == "minio" ]; then
   mkdir -p /app/objectstore/minio/
   echo "git" > /app/objectstore/minio/builder-bucket
```

Comment:
yes you can move it...it is only present in here as creating a file once should be enough...since you are moving it to `normalize_storage` script calling it once from the main boot file should be enough
To explain the exact thing happening here, https://github.com/deis/charts/blob/master/workflow-v2.3.0/tpl/deis-objectstorage-secret.yaml is the secret which gets templated based on the storage type used and is mounted on to the slugbuilder pod and if you see the minio section https://github.com/deis/charts/blob/master/workflow-v2.3.0/tpl/deis-objectstorage-secret.yaml#L36-L37 there is no bucket name to be used because we are hardcoding it to be `git` for builder and hence i am creating a file with contents as `git` so that the https://github.com/deis/object-storage-cli binary can download/upload it.